Media/MediaEvent/schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Media/MediaEvent/schema.json",
+  "title": "FIWARE - Media / Media Event",
+  "description": "Base for all events raised by elements in the Kurento media server",
+  "type": "object",
+  "allOf": [
+    { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons" },
+    {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "MediaEvent"
+          ],
+          "description": "NGSI Entity type"
+        },
+        "eventType": {
+          "type": "string"
```

Comment:
in general it is better to enumerate categories

Reply:
If possible, in this case, we prefer not to specify a fixed list